crypto/ec/ecp_nistp384.c

```diff
@@ -0,0 +1,1935 @@
+/*
+ * Copyright 2023 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/*
+ * ECDSA low level APIs are deprecated for public use, but still ok for
+ * internal use.
+ */
+#include "internal/deprecated.h"
```

Comment:
Why?  This is a new file, it should even mention deprecation.

Reply:
Well it could use some api internally that is deprecated, however I do not see any such api in use here.

Reply:
Apologies, this is residual from v1, where I had deprecation annotations on ossl_ec_nistp384_method().